### PR TITLE
Add Discord link to README

### PR DESCRIPTION
**Pull request by djc**
Will need to be merged to main to appear on the GitHub homepage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [![Chrono GitHub Actions][gh-image]][gh-checks]
 [![Chrono on crates.io][cratesio-image]][cratesio]
 [![Chrono on docs.rs][docsrs-image]][docsrs]
-[![Join the chat at https://gitter.im/chrono-rs/chrono][gitter-image]][gitter]
+[![Chat][gitter-image]][gitter]
```
Comment (pitdicker) [inline README.md:7]: I don't know just about anything about chat platforms since irc... Is gitter in some way related to Discord?

**Comment by pitdicker** [comment]
Thank you for setting this up!

File: README.md (hunk 1 above)
Comment (djc) [reply README.md:7]: Nope, they're unrelated. Gitter was acquired by Matrix at some point. There are ways to bridge Matrix and Discord but the bridges can be pretty flaky.

Comment (pitdicker) [reply README.md:7]: They `gitter` should be replaced with `discord`?

Comment (djc) [reply README.md:7]: Ahh, sorry, yes.

**Review by pitdicker — APPROVED** [review]

**Comment by pitdicker** [comment]
@jtmoon79 Also interested in joining the discord channel? (not meaning right now, but in general)

**Pull request merged**